tensorflow/python/ops/bincount_ops.py

```diff
@@ -215,6 +217,12 @@ def bincount(arr,
           binary_output=binary_output)
     else:
       weights = validate_dense_weights(arr, weights, dtype)
+      # check if we are using DenseBincount with GPU config
```

Comment:
Comments should be full sentences, capitalize the first letter and add punctuation.